boards/nxp/mcx_n9xx_evk/doc/index.rst

```diff
@@ -0,0 +1,294 @@
+MIPI_DBI interface of the shield, connected to the FlexIO on-chip peripheral.
+
+Dual Core samples
+*****************
+
++-----------+-------------------+----------------------+
+| Core      | Boot Address      | Comment              |
++===========+===================+======================+
+| CPU0      | 0x10000000[1856K] | primary core flash   |
++-----------+-------------------+----------------------+
+| CPU1      | 0x101d0000[192K]  | secondary core flash |
++-----------+-------------------+----------------------+
+
++----------+------------------+-----------------------+
+| Memory   | Address[Size]    | Comment               |
++==========+==================+=======================+
+| srama    | 0x20000000[320k] | CPU0 ram              |
++----------+------------------+-----------------------+
+| sramg    | 0x20050000[64k]  | CPU1 ram              |
++----------+------------------+-----------------------+
+| sramh    | 0x20060000[32k]  | Shared memory         |
++----------+------------------+-----------------------+
+
+Targets available
+==================
+
+The default configuration file
+:zephyr_file:`boards/nxp/mcx_n9xx_evk/mcx_n9xx_evk_mcxn947_cpu0_defconfig`
+only enables the first core. CPU0 is the only target that can run standalone.
+
+CPU1 does not work without CPU0 enabling it.
+
+To enable CPU1, create System Build application project and enable the
+second core with config :kconfig:option:`CONFIG_SECOND_CORE_MCUX`.
+
+Please have a look at some already enabled samples:
+
+- :zephyr:code-sample:`ipc-static-vrings`
+- :zephyr:code-sample:`openamp`
+- :zephyr:code-sample:`mbox`
+- :zephyr:code-sample:`mbox_data`
+
+Connections and IOs
+===================
+
+The MCX-N947 SoC has 6 gpio controllers and has pinmux registers which
+can be used to configure the functionality of a pin.
+
++------------+-----------------+----------------------------+
+| Name       | Function        | Usage                      |
++============+=================+============================+
+| P0_PIO1_8  | UART            | UART RX cpu0               |
++------------+-----------------+----------------------------+
+| P1_PIO1_9  | UART            | UART TX cpu0               |
++------------+-----------------+----------------------------+
+| P4_PIO4_3  | UART            | UART RX cpu1               |
++------------+-----------------+----------------------------+
+| P4_PIO4_2  | UART            | UART TX cpu1               |
++------------+-----------------+----------------------------+
+
+System Clock
+============
+
+The MCX-N947 SoC is configured to use PLL0 running at 150MHz as a source for
+the system clock.
+
+Serial Port
+===========
+
+The MCX-N9XX-EVK SoC has 10 FLEXCOMM interfaces for serial communication.
+Flexcomm 4 is configured as UART for the console.
+
+Ethernet
+========
+To use networking samples with the Ethernet jack, change jumper JP13 to pins 2-3.
+
+Programming and Debugging
+*************************
+
```

Comment:
Can definitely be a follow-up but would be nice to use the supported runners sphinx directive